auth/http.go

```diff
@@ -1405,6 +1430,57 @@ func httpHandlerGetGroups(central *ImqsCentral, w http.ResponseWriter, r *httpRe
 	}
 }
 
+func httpHandlerExportUserGroups(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
+
+	users, err := central.Central.GetAuthenticatorIdentities(authaus.GetIdentitiesFlagNone)
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	ident2perm, err := central.Central.GetPermits()
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	groups, err := central.Central.GetRoleGroupDB().GetGroupsRaw()
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	groupsResp, err := getPermitsJSON(central, users, ident2perm)
+	if err != nil {
+		authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+
+	var exportGroups []exportGroup
+	groupNametoID := make(map[string]int)
+	for _, group := range groups {
+		exportGroups = append(exportGroups, exportGroup{Id: int(group.ID), Name: group.Name, Permlist: group.PermList})
```

Comment:
I think it would look cleaner if he had a method on the group object called `ToExportGroup` or something.
The net result of how this line ought to read should be something like this:
```suggestion
		exportGroups = append(exportGroups, toExportGroup(group))
```

Reply:
I have to strongly disagree here, this nice is clear in its function and purpose